server/channels/app/platform/support_packet.go

```diff
@@ -266,3 +272,33 @@ func (ps *PlatformService) getGoroutineProfile(_ request.CTX) (*model.FileData,
 	}
 	return fileData, nil
 }
+
+// detectSAMLProviderType attempts to identify the SAML provider type based on the IdpDescriptorURL.
+// It returns "unknown" if the provider cannot be identified.
+func detectSAMLProviderType(idpDescriptorURL string) string {
+	if idpDescriptorURL == "" {
+		return unknownDataPoint
+	}
+
+	// Check for common SAML provider patterns in the EntityID/IdpDescriptorURL
+	switch {
+	case strings.Contains(idpDescriptorURL, "/realms/"):
```

Comment:
Where did you get this list and url patterns from?
And are you sure about these patterns like `/realms/` is not used by anyone but Keycloak?